src/config/networks/mainnet.ts

```diff
@@ -1,6 +1,5 @@
 import EtherLogo from 'src/config/assets/token_eth.svg'
 import { EnvironmentSettings, ETHEREUM_NETWORK, NetworkConfig } from 'src/config/networks/network.d'
-import { ETHGASSTATION_API_KEY } from 'src/utils/constants'
```

Comment:
Hey @mikheevm 

could you undo this change and fix here the ' -> ` change you added on PR #2564 ? 


Thank you for noticing

Reply:
Otherwise eslint will complain about unused variable

Reply:
I mean because here you are deleting the unused value but here you are fixing why that value was unused, so there will be a conflict once this two PRs are merged

https://github.com/gnosis/safe-react/pull/2564/files#diff-cc64278dd36991ff259260df6476637bfa638669dcb473a8ddb9cd8cc58778abR10

Reply:
Not sure if it's worth it, time-wise it would be the same to fix the conflict and remove the change : )